assets/admin/css/admin.css

```
@@ -372,6 +368,405 @@ textarea#episode_embed_code {
   font-weight: bold;
   margin: 5px 0 10px; }
 
+.ssp-admin {
+  background-color: #F1F5F9;
+  font-family: Roboto, -apple-system, BlinkMacSystemFont, "Segoe UI", Roboto, Oxygen-Sans, Ubuntu, Cantarell, "Helvetica Neue", sans-serif; }
```

Biome (2.1.2)

[error] Duplicate font names are redundant and unnecessary: Roboto

Remove duplicate font names within the property

(lint/suspicious/noDuplicateFontNames)